webrtc/lib/webrtc/variant_selector.ex

```diff
@@ -264,7 +287,7 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.VariantSelector do
       # and we have the bandwidth
```

Comment:
If we added here a new condition like below, would it mean that we don't have to introduce any changes in `best_active_variant` function?


```elixir
selector.muted_variant == variant ->
  {selector, action} = select_variant(selector, variant)
  action = add_reason(action, :some_reason)
  selector = manage_allocation(selector)
  {selector, action}
```

Reply:
Looks great, I will apply this change